packages/components/organisms/f-content-cards/src/components/cardTemplates/StampCard1.vue

```diff
@@ -67,11 +77,35 @@
 
 <script>
 import parseISO from 'date-fns/parseISO';
-import lightFormat from 'date-fns/format';
+import format from 'date-fns/format';
+import lightFormat from 'date-fns/lightFormat';
 
 import EmptyStamp from './images/stamp-empty-15.svg';
 import FullStamp from './images/stamp-full-15.svg';
 
+import '../MakeTextAccessible';
+
+const getDateFnsLocale = locale => {
+    switch (locale) {
+        case 'da-DK':
+            return import(/* webpackChunkName */ 'date-fns/locale/da');
```

Comment:
What do the generated bundles look like when lazy-loading? When I've tried doing this in the past I think the UMD bundle ended up with everything included.

Reply:
![Screenshot 2021-03-04 at 11 57 11](https://user-images.githubusercontent.com/15876339/109960503-c3893600-7ce0-11eb-96c8-b72cfba0dbd3.png)

It separates the locales out ok

Reply:
each one with number is just the date code for that locale

Reply:
Nice, I'd be interested to see this in action when used in a project as I tried and failed to get this working in the past.